src/utils/api.js

```diff
@@ -10,8 +12,9 @@ APIError.prototype = Object.create(Error.prototype);
 APIError.prototype.name = 'APIError';
 
 export default class APIClient {
-  constructor(root = 'https://localhost:8000/api/') {
+  constructor(root = NORMANDY_API_ROOT_URL, accessToken = null) {
     this.root = root;
+    this.accessToken = accessToken;
```

Comment:
Pardon my ignorance of this API but it feels really weird to make the accessToken part of the constructor and not the `fetch` method. Now every single use of the APIClient is either going to be all with-accesstoken or all without-accesstoken.